lib/request.js

```diff
@@ -8,7 +8,7 @@ async function request(query, variables) {
     uri: 'https://api.github.com/graphql',
     method: 'POST',
     headers: {
-      'Authorization': `Basic ${auth}`,
+      'Authorization': `Basic ${await auth()}`,
```

Comment:
I think we should do this during start up instead of reading/prompting for crendentials everytime a request is sent?

Reply:
@joyeecheung Ah, right, I *meant* to cache the value here :smile: